### PR TITLE
¡Hola, 🇪🇸!

### PR DESCRIPTION
**Pull request by Tastaturberuf**

### CODE REVIEW

File: meta/tastaturberuf/contao-image-copyright-bundle/es.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+es:
+    title: Información sobre la licencia de las imágenes
+    description: >
+        Con este paquete de Contao, puede añadir fácilmente un aviso de licencia para cada imagen y luego mostrarlas en un módulo.
+        Especialmente útil si utiliza fotos de stock de varios proveedores y necesita mostrar la información de la licencia.
```
Comment (fritzmg) [inline meta/tastaturberuf/contao-image-copyright-bundle/es.yml:5]: `stock` needs to be added to the `es` allow list.

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: meta/tastaturberuf/contao-image-copyright-bundle/es.yml (hunk 1 above)
Comment (Tastaturberuf) [reply meta/tastaturberuf/contao-image-copyright-bundle/es.yml:5]: All checks have passed

**Comment by Tastaturberuf** [comment]
ich habe einfach keinen bock hier 10 commits zumachen. das linten is eh
sinnlos.

Fritz Michael Gschwantner ***@***.***> schrieb am Mo., 2.
Aug. 2021, 23:09:

**Comment by fritzmg** [comment]
All you had to do was adding `stock` to the `es` allow list. The linter is there to help you detect errors in your meta data description.

However, the Spanish translation for "stock fotos" seems to be _banco de imágenes_. That's what Pixabay uses on its Spanish version as well. I have made the necessary changes directly.

**Review by fritzmg — APPROVED** [review]

**Comment by Toflar** [comment]
> ich habe einfach keinen bock hier 10 commits zumachen. das linten is eh sinnlos.

Nein. Ist es nicht.

**Pull request merged**